build/build.sh

```diff
@@ -51,7 +51,7 @@ pandoc --verbose \
   $INPUT_PATH
 
 # Return null if docker command is missing, otherwise return path to docker
-DOCKER_EXISTS=`command -v docker`
+DOCKER_EXISTS="$(command -v docker || true)"
```

Comment:
So this is saying that if the `command -v docker` returns null, then set `DOCKER_EXISTS=true`? Seems like this will always make `DOCKER_EXISTS` not null and therefore trigger the docker PDF build below?

Reply:
@dhimmel there is some relevant discussion here: https://unix.stackexchange.com/questions/325705/why-is-pattern-command-true-useful/325727
> Rephrased: it masks the error status of the previous command.

In some quick testing with Git Bash on Windows, this is needed for a script with `set -o errexit`.
```
DOCKER_EXISTS=`command -v docker`
```
stops the script if `docker` isn't available, as reported in #229.

Reply:
Okay here is what I get using bash on Linux:

```
$ DOCKER_EXISTS=`command -v docker || true`
$ echo $DOCKER_EXISTS
/usr/bin/docker
$ DOCKER_EXISTS=`command -v docker-mispelled || true`
$ echo $DOCKER_EXISTS

$ DOCKER_EXISTS=`command-mispelled -v docker || true`
command-mispelled: command not found
$ echo $DOCKER_EXISTS

```

So it does seem that `|| true` has the intended effect of silencing the error.

Is the switch from backticks to `$()` for command substitution required for windows bash, or is this a preference, as per [this discussion](https://stackoverflow.com/a/24592916/4651668)?

Reply:
I wonder if we should use:

```sh
DOCKER_EXISTS=$(docker --version || true)
```

Because if the `docker` command exists, but the `command` command doesn't, we still want to make `DOCKER_EXISTS` non-null.

Reply:
@dhimmel: `$(..)` was originally introduced in ksh, and later bash and POSIX borrowed this idea because it was better to read and nest other commands inside. I rarely used this format because it used to be incompatible with classic shell. But since it is now in POSIX, it is much safer now.

Reply:
Using `docker --version` instead of `command` does seem more direct and relevant here.

Reply:
When using `DOCKER_EXISTS=$(docker --version || true)` on a system that doesn't have docker installed, you may get error messages in stderr like this:
```
 Command 'docker' not found, but can be installed with:

sudo snap install docker     # version 18.06.1-ce, or
sudo apt  install docker.io

See 'snap info docker' for additional versions.
```
This is distracting. You have to hide it by this:
`DOCKER_EXISTS=$(docker --version > /dev/null 2>&1 || true)`
This may not be what you like.

`command` is actually a bash built-in:
https://www.gnu.org/software/bash/manual/html_node/Bash-Builtins.html#Bash-Builtins